src/sentry/nodestore/django/backend.py

```diff
@@ -13,21 +14,39 @@
 class DjangoNodeStorage(NodeStorage):
     def delete(self, id):
         Node.objects.filter(id=id).delete()
+        self._delete_cache_item(id)
 
     def get(self, id):
+        item_from_cache = self._get_cache_item(id)
+        if item_from_cache:
+            return item_from_cache
         try:
-            return Node.objects.get(id=id).data
+            data = Node.objects.get(id=id).data
```

Comment:
Don't we have a separate mechanism to cache DJango models ?
See BaseManager

Reply:
We do, however I'd like to keep this implementation similar to the bigtable one (and using the new cache). I don't expect open source users who are using the Django backend are likely to use this, it's mostly targeted at bigtable performance issues.

Reply:
I would reconsider this. We can take it as a separate PR, if this is for big table, then let's not provide an implementation for postgres we have to maintain. If instead we want to have a postgres cache we go back to my initial quesiton, why don't we use the caching system we already have in the BaseManager

Reply:
I need to look at Postgres caching again to answer your question properly. The reason I've kept them the same was mostly to get some test coverage and ease of development since Bigtable doesn't run in CI and I had to skip those tests. I haven't thought too much about what we want to provide people using this backend in production yet.